src/Analyzer.Reports.UnitTests/ConsoleReportWriterTests.cs

```diff
@@ -0,0 +1,586 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Abstractions;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Azure.Templates.Analyzer.Reports.UnitTests
+{
+    [TestClass]
+    public class ConsoleReportWriterTests
+    {
+        [TestMethod]
+        public void ConsoleReportWriter_SingleEvaluationSingleFailedResult()
```

Comment:
For the test names we follow this pattern: {method name}\_{what is being tested}\_{expected outcome} https://github.com/Azure/template-analyzer/blob/main/CONTRIBUTING.md#tests

Reply:
fixed